### DISCUSSION TITLE
Is there a context menu command to silence the current browser?

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#3595] [Q&A]
Is there a context menu command to silence the current browser implementation in winform?

**Comment by amaitland** [comment]
Add your own custom menu item.

http://cefsharp.github.io/api/90.6.x/html/M_CefSharp_IBrowserHost_SetAudioMuted.htm
http://cefsharp.github.io/api/90.6.x/html/P_CefSharp_IBrowserHost_IsAudioMuted.htm

**Reply by wnunezc** [reply]
@amaitland the version still not reaching the nuget package manager i only see 89.xx

**Reply by amaitland** [reply]
https://www.nuget.org/packages/CefSharp.WinForms/90.6.70

IsAudioMuted is not a new feature, it was added in version 73.
